### DISCUSSION TITLE
Chunked encoding, joined AV1 files and RPU metadata

### DISCUSSION THREAD

**Discussion by Boulder08** [quietvoid/dovi_tool#289] [General]
I've been experimenting with your SVT-AV1 patch to support DoVi in AV1. Single files work fine on Kodi and trigger DoVi mode on my TV. However, to speed up encoding, I'm thinking of running a parallel chunked encoding method where the original RPU file is split based on the chunks and each separate RPU input during the encode, and then the encoded parts would be joined to a single mkv file using mkvmerge.

Would there be any DoVi related issues in this case? Since there's no tool for extracting the RPU out of the final AV1 track, I cannot check how it looks compared to the original one and I'm not sure how to really verify this kind of a thing during playback.

**Comment by quietvoid** [comment]
It should be fine.

There are some test files that you can try chunking and see if the RPU has the expected effect before/after encoding.
Such as https://mega.nz/file/4BUihYTB#3CRDt-5MpVkX1NB0Mz9ZH4gEFTRTog4hxRR39URVTNU
